backend/attendance-service/src/main/java/org/egov/validator/AttendanceLogServiceValidator.java

```diff
@@ -71,21 +71,23 @@ public void validateUpdateAttendanceLogRequest(AttendanceLogRequest attendanceLo
     private void validateTenantIdAssociationWithRegisterId(AttendanceLogRequest attendanceLogRequest) {
         String tenantId = attendanceLogRequest.getAttendance().get(0).getTenantId();
         String registerId = attendanceLogRequest.getAttendance().get(0).getRegisterId();
-        validateTenantIdAssociationWithRegisterId(tenantId,registerId);
+        validateTenantIdAssociationWithRegisterId(tenantId, registerId);
     }
+
     private void validateDocumentIds(AttendanceLogRequest attendanceLogRequest) {
-        if("TRUE".equalsIgnoreCase(config.getDocumentIdVerificationRequired())){
+        if ("TRUE".equalsIgnoreCase(config.getDocumentIdVerificationRequired())) {
             //TODO
             // For now throwing exception. Later implementation will be done
-            throw new CustomException("SERVICE_UNAVAILABLE","Service not integrated yet");
+            throw new CustomException("SERVICE_UNAVAILABLE", "Service not integrated yet");
         }
     }
+
     private void validateAttendanceLogIds(AttendanceLogRequest attendanceLogRequest) {
         List<AttendanceLog> attendance = attendanceLogRequest.getAttendance();
-        List<String> providedAttendanceLogIds = attendance.stream().map(e->String.valueOf(e.getId())).collect(Collectors.toList());
+        List<String> providedAttendanceLogIds = attendance.stream().map(e -> String.valueOf(e.getId())).collect(Collectors.toList());
```

Comment:
Why do we need to do `String.valueOf()`, if id is already a String. If it isn't, please make that attribute type to be String.

Reply:
Changed to accept String datatype instead of UUID